pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuild.java

```diff
@@ -173,7 +170,7 @@ public void downloadArtifact(String pattern, File downloadedZip) {
         }
     }
 
-    private List<String> readLog(InputStream inputStream) throws IOException {
+    private static List<String> readLog(InputStream inputStream) throws IOException {
```

Comment:
Does it need to be static?

Reply:
I suppose nothing _needs_ to be `static`, but IMO if can, then it should be. Sometimes, static method may indicate a design problem, though.